### PR TITLE
chore: remove test values from application.yml

### PR DESCRIPTION
**Pull request by cromoteca**

### CODE REVIEW

File: sso-kit-hilla-demo/src/main/resources/application.yml [hunk 1]
```diff
@@ -13,25 +13,7 @@ spring:
     defer-datasource-initialization: true
```
Comment (tamasmak) [inline sso-kit-hilla-demo/src/main/resources/application.yml:13]: Do we still need this configuration?

File: sso-kit-hilla-demo/src/main/resources/application.yml [hunk 2]
```diff
@@ -13,25 +13,7 @@ spring:
...
   mustache:
     check-template-location: false
```
Comment (tamasmak) [inline sso-kit-hilla-demo/src/main/resources/application.yml:15]: Same for this. Do we need this?

File: sso-kit-hilla-demo/src/main/resources/application.yml (hunk 1 above)
Comment (cromoteca) [reply sso-kit-hilla-demo/src/main/resources/application.yml]: Removed

File: sso-kit-hilla-demo/src/main/resources/application.yml (hunk 2 above)
Comment (cromoteca) [reply sso-kit-hilla-demo/src/main/resources/application.yml]: Removed

**Review by tamasmak — APPROVED** [review]

**Pull request merged**